airbyte-ci/connectors/pipelines/pipelines/telemetry.py

```diff
@@ -0,0 +1,42 @@
+import getpass
+import os
+import sys
+
+import segment.analytics as analytics
+
+analytics.write_key = "G6G7whgro81g9xM00kN2buclGKvcOjFd"
+analytics.send = True
+analytics.debug = True
+
+
+def _is_airbyte_user():
+    return os.getenv("AIRBYTE_ROLE") == "airbyter"
+
+
+def track_command(f):
+    """
+    Decorator to track CLI commands with segment.io
+    """
+
+    def wrapper(*args, **kwargs):
+        ctx = args[0]
+        top_level_command = ctx.command_path
+        full_cmd = " ".join(sys.argv)
+
+        sys_user_name = getpass.getuser()
```

Comment:
Any PII worries here?  I guess this is an internal tool for now

Reply:
Good callout. @evantahler Ill go ahead and hash their machine info